scipy/stats/_hypotests.py

```diff
@@ -956,6 +958,204 @@ def barnard_exact(table, alternative="two-sided", pooled=True, n=32):
     return BarnardExactResult(wald_stat_obs, p_value)
 
 
+BoschlooExactResult = make_dataclass(
+    "BoschlooExactResult", [("statistic", float), ("pvalue", float)]
+)
+
+
+def boschloo_exact(table, alternative="two-sided", n=32):
+    r"""Perform a Boschloo exact test on a 2x2 contingency table.
```

Comment:
```suggestion
    r"""Perform Boschloo's exact test on a 2x2 contingency table.
```
When I search for "Boschloo exact test", I only see results for "Boschloo's exact test". This suggests that "Boschloo's exact test" is much more common.